src/testing/README.md

```diff
@@ -0,0 +1,60 @@
+## VOPR Output
+
+### Columns
+
+1. Replica index.
+2. Event:
+    - `$`: crash
+    - `^`: recover
+    - ` `: commit
+    - `[`: checkpoint start
+    - `]`: checkpoint done
+3. Role (according to the replica itself):
+    - `/`: primary
+    - `\`: backup
+    - `|`: standby
+    - `#`: (crashed)
```

Comment:
These looks a bit mysterious. Maybe `P b s c` / `P B S X` ? Ahhh, I think we want to avoid alphanumerics to not clash with the hash, right?

Reply:
looking at the actual output, the aesthetics of \|/ is just too good :D